src/test/java/net/sf/jabref/importer/fileformat/MedlineImporterTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *
+ */
+package net.sf.jabref.importer.fileformat;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+/**
+ * @author Daniel Mair/Bruehl
+ *
+ */
+public class MedlineImporterTest {
+
+    MedlineImporter medlineImp;
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+        this.medlineImp = new MedlineImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormatAccept() throws Throwable {
+        List<String> list = Arrays.asList("MedlineImporterTest.xml");
+        for (String str : list) {
+            try (InputStream is = MedlineImporterTest.class.getResourceAsStream(str);) {
+                Assert.assertTrue(medlineImp.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatReject() throws Throwable {
+        List<String> list = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt",
+                "IEEEImport1.txt", "IsiImporterTestWOS.isi");
+        for (String str : list) {
+            try (InputStream is = MedlineImporterTest.class.getResourceAsStream(str);) {
+                Assert.assertFalse(medlineImp.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testImportEntries() throws IOException {
+        try (InputStream is = MedlineImporter.class.getResourceAsStream("MedlineImporterTest.xml")) {
+            List<BibEntry> entries = medlineImp.importEntries(is, new OutputPrinterToNull());
+            Assert.assertEquals(1, entries.size());
```

Comment:
This is not a good assertion, you need to verify as well all the fields of the BibEntry.